plugins/commands/create.js

```diff
@@ -1,10 +1,72 @@
+/* eslint no-underscore-dangle: ["error", { "allow": ["_data", "_shot"] }] */
+
 'use strict';
 
 const boom = require('boom');
 const schema = require('screwdriver-data-schema');
 const validator = require('screwdriver-command-validator');
 const hoek = require('hoek');
 const urlLib = require('url');
+const req = require('request');
+const VERSION_REGEX = schema.config.regex.VERSION;
+
+/**
+ * Publish binary command
+ * @method binaryCommandPublish
+ * @param  {commandFactory} commandFactory      commandFactory
+ * @param  {config}         config              Command config
+ * @param  {binary}         binary              Binary published to the store
+ * @param  {string}         storeUrl            URL to the store
```

Comment:
Can you capitalize these (aka `String`, `Binary`, etc)?

Reply:
Capitalized these.